### DISCUSSION TITLE
REST API endpoint for get model signatures?

### DISCUSSION THREAD

**Discussion by mkredpoint** [mlflow/mlflow#5772] [Q&A] [answered]
I've been scouring the MLFlow rest api documentation (https://www.mlflow.org/docs/latest/rest-api.html) but I don't see any endpoints that provide input/output signature information about registered models. Is there any way to programmatically determine what order,types, etc of inputs a registered model expects using REST?

**Comment by BenWilson2** [comment, marked as answer]
We don't support that (as of yet) but that's a really great idea. Could I trouble you to file a FR issue that describes the mechanism that you think would work best for your use case?
